bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -32,10 +32,13 @@ java_proto_library(
     deps = [":{{name}}_proto"],
 )
 
-java_grpc_library(
-    name = "{{name}}_java_grpc",
-    srcs = [":{{name}}_proto"],
-    deps = [":{{name}}_java_proto"],
+# Open Source Packages
+java_gapic_assembly_gradle_pkg(
+    name = "{{type_only_assembly_name}}-java",
```

Comment:
This is consistent with the rest of the file so I think it's good in the context of the PR. 
However, do we want to keep the it consistent with the names of regular packages? I don't see anything wrong with `google-cloud-alloydb-connectors-v1alpha-java` for types only packages. It may not matter for Bazel bot but the benefit I see is that 

1.  Developers do not have to be aware of the differences between type-only and a regular package.
2.  It's a more predictable name when we want to test it locally.

Reply:
Type only packages don't always have a version in them, sometimes they are just `google/shopping/type`. In this case it would like `google-shopping-type-java` which isn't bad either.

Why don't we just use this name `type_only_assembly_name` variable here, and I will change the code in a separate PR to update the format of its value. It will impact other languages as well so best to keep in a separate PR.

Reply:
Yes, agree that it should be a different PR even if we want to change it.